src/styles/index.ts

```diff
@@ -2912,6 +2917,15 @@ const styles = (theme: ThemeColors) =>
             marginRight: 20,
         },
 
+        dividerLine: {
```

Comment:
Similar question - don't we already have divider lines in the product? Maybe not, but just checking to see if we can reuse an existing style if possible. If not, then no worries!